### PR TITLE
Coverage for Bug 2128263

### PR DESCRIPTION
**Pull request by vkathole**
Check that alert KubePersistentVolumeInodesFillingUp is not firing.
Signed-off-by: vkathole <vkathole@redhat.com>

### CODE REVIEW

File: tests/e2e/workloads/app/test_jenkins_simulation.py [hunk 1]
```diff
@@ -82,4 +84,19 @@ def test_git_clone(self, pod):
         ), f"Logs should not contain the error message '{error_msg}'"
         logger.info(f"Logs did not contain the error message '{error_msg}'")
 
+        # Test coverage for Bz 2128263
```
Comment (PrasadDesala) [inline tests/e2e/workloads/app/test_jenkins_simulation.py:87]: As per the BZ we need to reconfigure OpenShift Container Platform registry to use RWX, that complete step is missing. Also please move/create a test under registry. This is not a relevant location

File: tests/e2e/workloads/ocp/registry/test_registry_by_increasing_num_of_image_registry_pods.py [hunk 2]
```diff
@@ -71,6 +75,20 @@ def test_registry_by_increasing_num_of_registry_pods(self, count=3):
...
+        )
+        alerts = api.wait_for_alert(
+            name=constants.ALERT_KUBEPERSISTENTVOLUMEINODESFILLINGUP,
+            timeout=100,
+            sleep=1,
+        )
+        if len(alerts) > 0:
+            assert (
+                False
+            ), f"Failed: There should be no {constants.ALERT_KUBEPERSISTENTVOLUMEINODESFILLINGUP} alert"
```
Comment (PrasadDesala) [inline tests/e2e/workloads/ocp/registry/test_registry_by_increasing_num_of_image_registry_pods.py:91]: do we need to also check for kubelet_volume_stats_inodes in csi-cephfsplugin pod logs ?

File: tests/e2e/workloads/ocp/registry/test_registry_by_increasing_num_of_image_registry_pods.py [hunk 3]
```diff
@@ -71,6 +75,20 @@ def test_registry_by_increasing_num_of_registry_pods(self, count=3):
 
         # Validate pvc mounted on image registry pod
         validate_pvc_mount_on_registry_pod()
+        # Coverage for Bz 2128263
```
Comment (PrasadDesala) [inline tests/e2e/workloads/ocp/registry/test_registry_by_increasing_num_of_image_registry_pods.py:78]: You are doing this verification at the very beginning and no registry ops were done here.
Should we move this coverage to the end?

File: tests/e2e/workloads/ocp/registry/test_registry_by_increasing_num_of_image_registry_pods.py [hunk 4]
```diff
@@ -84,3 +90,29 @@ def test_registry_by_increasing_num_of_registry_pods(self, count=3):
 
         # Validate image registry pods
         validate_registry_pod_status()
+
+        # Coverage for Bz 2128263
```
Comment (avd-sagare) [inline tests/e2e/workloads/ocp/registry/test_registry_by_increasing_num_of_image_registry_pods.py:94]: reconfigure OpenShift Container Platform registry to use RWX, this complete step is missing

**Review by PrasadDesala — APPROVED** [review]

**Review by avd-sagare — APPROVED** [review]

**Pull request merged**